### ISSUE TITLE
User accuracy should not incorporate AI validations

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#4402]
User accuracy shown on explore, dashboard, leaderboard, etc. should not incorporate AI validations. AI is simply not as good (yet) as humans. And we need better ways of tracking AI performance before we can incorporate it like this into key stats.

**Comment by misaugstad** [comment]
My (anecdotal) experience is telling me that AI has been about as good as humans (on average). And we're using a confidence threshold that matched a 92% precision on the test set when training the model.

I agree that we should do more testing to determine how well it's doing, but I thought that we had it set up in a way where we could be reasonably confident that it was doing a good job (at least in cities that are present in the training set).

Here's the repo with the validator: https://github.com/ProjectSidewalk/sidewalk-validator-ai

**Comment by jonfroehlich** [comment]
Good to know. We should definitely have a way of tracking this and training on incoming data as well, of course, which we have long talked about.
